index.js

```diff
@@ -22,3 +22,5 @@ Object.defineProperty(exports, 'test', {
 });
 
 exports.Transaction = require('loopback-connector').Transaction;
```

Comment:
I have no idea, whether `Transaction` is needed or not. IMO, this is unrelated to the key-value implementation and therefore we should not change it in this pull request.

Removing `exports.Transaction` is also a breaking change.

It makes sense to remove it in 3.0, since users should access `Transaction` directly from `loopback-connector`.

@raymondfeng would you mind opening a pull request for this change yourself? Just don't forget to update 3.0-RELEASE-NOTES.md